docker-compose.yml

```diff
@@ -24,6 +24,10 @@ services:
       - db
       - mailcatcher
     env_file: local.env
+    environment:
+      VIRTUAL_HOST: demo-bridge-uof.bayesimpact.org
+      LETSENCRYPT_HOST: demo-bridge-uof.bayesimpact.org
+      LETSENCRYPT_EMAIL: pascal@bayesimpact.org
```

Comment:
Since this is the main global repo for all hypothetical deployments of bridge-uof, I don't think we should have bayes specific env variables defined in a checked-in file. For the bayes-hosted demo, we just have the env variable files on the instance itself. For the deployments with DOJ, we have a separate repo (https://github.com/bayesimpact/ursus-deployment).

I went ahead and set your variables on the Bayes demo site and restarted it, so it should be ready.